File: Carrefour.Web/wwwroot/Scripts/Models/Product.js
class Product extends Entity
{
    constructor(json)
    {
        super(json);

        if (json)
        {
            this.Nameproduct = json.nameproduct;
            this.Category = json.category;
            this.Price = json.price;
            this.From = json.from;
            this.Gluten = json.gluten;
            this.Lactosa = json.lactosa;
        }

        else
        {
            this.Nameproduct = "";
            this.Category = "";
            this.Price = 0;
            this.From = "";
            this.Gluten = "";
            this.Lactosa = "";
        }
    }
}

File: Carrefour.lib/Models/Product.cs
namespace Carrefour.lib.Models
{

    public class Product : Entity
    {
        public string Nameproduct { get; set; }
        public string Category { get; set; }
        public int Price { get; set; }
        public string From { get; set; }
        public bool Gluten { get; set; }
        public bool Lactosa { get; set; }
    }
    
}
